src/components/templates/PartyMap/components/Map/MapRoom.scss

```diff
@@ -37,6 +39,22 @@ $label-visible: flex;
     }
   }
 
+  &--unclickable {
+    cursor: default;
+
+    &:hover {
+      transform: none;
+
+      .maproom__label {
+        display: $label-hidden;
+      }
```

Comment:
This won't be needed if you conditionally render that section, as mentioned in another comment in this review.